bindings/imap-codec-python/Cargo.toml

```diff
@@ -13,4 +13,6 @@ crate-type = ["cdylib"]
 doc = false
 
 [dependencies]
+imap-codec = { version = "2.0.0-alpha.2", path = "../../imap-codec", features = ["serde"] }
```

Comment:
Do we want to enable all other features, too? (Similar to [`imap-next`](https://github.com/duesee/imap-next/blob/829617afe42a6987832c132f3bc52a3177dfb9d6/Cargo.toml#L22C43-L22C51))

Reply:
I had the same thought but hesitated... It feels a bit off that these are features that almost everybody wants. I saw it a bit on my plate to finally make them on-by-default (inline). But... maybe we should better add them here. Otherwise there is no option to activate them.

Reply:
From `imap-next`:
```toml
[features]
default = ["stream"]
# ...
[dependencies]
imap-codec = { version = "2.0.0-alpha.2", features = ["starttls", "quirk_crlf_relaxed", "ext_condstore_qresync", "ext_login_referrals", "ext_mailbox_referrals", "ext_id", "ext_sort_thread", "ext_binary", "ext_metadata", "ext_uidplus"] }
```
If these are the features "almost everybody wants", it really sounds as like we should reconsider the list of default features.

It is probably a good point to think about how be able to control quirks/extensions from Python at some point. -> #547

Anyhow, which features should I enable for the Python bindings for now - exactly the list from `imap-next`?

Reply:
Yes. I filed https://github.com/duesee/imap-codec/issues/548

Let's not block on that here. I think it's more important to merge this PR. I can take of the features when tackling the issue.

Reply:
Okay, then I will leave it set to `default` + `serde` and it will extend automatically when extending `default`.